### DISCUSSION TITLE
Warnings baseline: report only new issues for existing code base

### DISCUSSION THREAD

**Discussion by AlexTrotsenko** [LemonAppDev/konsist#1297] [Request a Feature] [answered]
### Tell us more.

I would be nice to use konsist on the existing projects, which some legacy code is present.

The behaviour and set-up should be similar to Android lint:
https://developer.android.com/studio/write/lint#snapshot

While having gradle sub-modules could be a work-around to this issue, it's complicated and not always possible if specific sub-module structure exists already.

**Comment by igorwojda** [comment, marked as answer]
Baseline support should be available soon.

**Reply by AlexTrotsenko** [reply]
Thanks for the update! Do you know which version should/might contain this feature ?

**Reply by igorwojda** [reply]
0.17.0 or 0.18.0

**Reply by igorwojda** [reply]
Coming in 0.18.0
